templates/eng-workflow/build/build.1.schema.json

```diff
@@ -0,0 +1,149 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#", 
```

Comment:
nit: please change the indentation in this file to use 2 spaces per level.

Reply:
Fixed. :)